opshin/tests/test_Unions.py

```diff
@@ -0,0 +1,242 @@
+import unittest
+
+import hypothesis
+from hypothesis import given
+from hypothesis import strategies as st
+from .utils import eval_uplc_value
+from . import PLUTUS_VM_PROFILE
+from opshin.util import CompilerError
+
+hypothesis.settings.load_profile(PLUTUS_VM_PROFILE)
+
+from .test_misc import A
+
+
+def to_int(x):
+    if isinstance(x, A):
+        return 5
+    elif isinstance(x, int):
+        return 6
+    elif isinstance(x, bytes):
+        return 7
+    elif isinstance(x, list):
```

Comment:
In order to make really sure that Python and OpShin map to the same operaton here (i.e. that List is allowed for isinstance)
```suggestion
    elif isinstance(x, List):
```